src/ex2/htmlEditor.js

```diff
@@ -0,0 +1,136 @@
+class HtmlEditor {
+  constructor(_itemManager, _pokemonClient) {
+    this.img_src = "./images/delete_icon.svg";
+    this.li_class = "list-item";
+    this.img_class = "list-item-delete-button";
+    this.itemManager = _itemManager;
+    this.pokemonClient = _pokemonClient;
+    this.ul = document.getElementById("list");
+    this.emptyTasks = true;
+    this.clear_all_button;
+    this.sortArrow = document.getElementById("sort_arrow");
+    this.sortDirection = "up";
+  }
+
+  setDisplayEmptyTasks() {
+    this.changeDIsplayOfElement("none", "buttom_bar_container");
+    this.changeDIsplayOfElement("none", "sort");
+    this.emptyTasks = true;
+  }
+
+  createNewTag(tagName) {
+    return document.createElement(tagName);
+  }
+
+  setLiTag(li, text) {
+    li.id = this.pokemonClient.getPokemonIdByName(text);
```

Comment:
The html editor should not be aware of any http client, it should only edit html.
making a call here to pokemonClient breaks the separation of concerns.

htmlEditor should use itemManager. (itemManager might use pokemonClient)